client/src/curriculum/index.scss

```diff
@@ -289,9 +289,15 @@
       }
 
       scrim-inline {
+        aspect-ratio: 1.5;
         display: block;
-        height: 14.25rem;
-        width: 22rem;
+        margin: 0.5rem auto;
+        max-width: 36rem;
+        width: 100%;
+      }
+
+      p:has(> scrim-inline:only-child) {
+        margin: 1rem 0;
```

Comment:
Do we really need this. Starting to use `:has` would be a the first time here.

Reply:
We don't *need* it, but it's a small visual improvement, which is exactly what `:has` is perfect for.

Without:

![image](https://github.com/user-attachments/assets/0ddd7e7c-38ee-41ac-bf06-c418c44c3dc1)

With:

![image](https://github.com/user-attachments/assets/164ea733-ab63-470c-8374-e0e9a78580a5)